_posts/2018-11-07-arc-gis-server-10-6-upgrade.md

```diff
@@ -10,12 +10,12 @@ categories:
 tags: []
 ---
 
-AGRC is planning to upgrade the mapserv ArcGIS Server system to version 10.6.1 within the next few weeks. For the majority of our users, we do not expect any issues as _most_ services are being migrated as is. _However_, if you are consuming Image Services, **a migration is necessary** since AGRC is deprecating Image Services. 
+AGRC is planning to upgrade the mapserv ArcGIS Server system to version 10.6.1 within the next few weeks. For the majority of our users, we do not expect any issues, as _most_ services are being migrated as is. _However_, if you are consuming image services, **a migration is necessary** since AGRC is deprecating image services. 
 
-### AGRC is deprecating the Image Services on mapserv.utah.gov
+### AGRC Is Deprecating the Image Services on mapserv.utah.gov
 {: .pop}
 
-{% capture contact %}{% include contact.html subject=page.title contact=site.data.contacts.change_manager text="If you notice any other issues after the upgrade, please let" hide-punctuation=true%}{% endcapture %}
+{% capture contact %}{% include contact.html subject=page.title contact=site.data.contacts.change_manager text="If you notice any issues after the upgrade, please let" hide-punctuation=true%}{% endcapture %}
 
```

Comment:
One more thing I forgot to comment on: 
![image](https://user-images.githubusercontent.com/36281700/48276516-6bd6aa80-e405-11e8-90d3-a078c4e1eaf1.png)
It looks like there's quite a bit of space between Migration Options and Local Mosaic Datasets--is this how it's supposed to be? Same thing before Services Marked for Deprecation.